ci/build/build-code-server.sh

```diff
@@ -3,7 +3,6 @@ set -euo pipefail
 
 # Builds code-server into out and the frontend into dist.
 
-# MINIFY controls whether parcel minifies dist.
 MINIFY=${MINIFY-true}
```

Comment:
I don't know if we need this?

Also not sure if this means we need to add [`tinyify`](https://www.npmjs.com/package/tinyify) (browserify plugin for optimizing production bundles)

Reply:
True, it might make sense to add that so our bundles are optimized.